test/promql/tests/data_plane/release_service_test.yaml

```diff
@@ -57,3 +57,24 @@ tests:
               alert_team_handle: <!subteam^S03SVBS426R>
               runbook_url: https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/release-service/release-validation-latency.md?ref_type=heads
               team: release
+
+  - interval: 1m
```

Comment:
I suggest adding a test case that will include data series from multiple clusters (i.e. entries with `source_cluster` with varying values). This way the tests can verify that we aggregate what we wanted to aggregate. E.g. you might not want to aggregate time series between different clusters.

Reply:
Indeed. It makes  total sense.

Reply:
done.

Reply:
I think this is currently summing across all clusters. Is that the intention?

Reply:
yes. We consider the service down if the pod is down in any cluster.